src/main/java/chess/domain/position/Row.java

```diff
@@ -0,0 +1,41 @@
+package chess.domain.position;
+
+import chess.exception.InvalidRowException;
+
+import java.util.Arrays;
+
+public enum Row {
+    EIGHTH("8", 8), SEVENTH("7", 7), SIXTH("6", 6), FIFTH("5", 5),
+    FOURTH("4", 4), THIRD("3", 3), SECOND("2", 2), FIRST("1", 1);
+
+    private final String lineName;
+    private final int value;
+
+    Row(String lineName, int value) {
+        this.lineName = lineName;
+        this.value = value;
+    }
+
+    public static Row from(Character lineName) {
+        return Arrays.stream(Row.values())
+                .filter(Row -> Row.lineName.equals(lineName.toString()))
```

Comment:
```suggestion
                .filter( row -> row.lineName.equals(lineName.toString()))
```
자바에서 변수는 소문자로 시작하는 것이 관례입니다.

Reply:
아이고 이런 실수를...ㅜ 수정하겠습니다!